homeassistant/components/automation/time.py

```diff
@@ -27,7 +28,7 @@ def time_automation_listener(now):
         hass.async_run_job(action, {
             'trigger': {
                 'platform': 'time',
-                'now': now,
+                'now': dt_util.as_local(now),
```

Comment:
This is not the right fix. `async_track_time_change` operates on local time and so `now` should already be local. If it's not, we should fix that inside `helpers/event.py`.

Reply:
OK, now fixed inside `helpers/event.py` instead